tests/cicd/main_build.yml

```diff
@@ -351,3 +351,26 @@ jobs:
       pytest -s tests/cicd/src/notebook_integration_tests.py \
         --nbname examples/interpretation/segyconverter/02_segy_convert_sample.ipynb \
         --cwd examples/interpretation/segyconverter
+
+
+###################################################################################################
+# Stage 5: Docker tests
+###################################################################################################
+
+
+- job: docker_build_test
+  dependsOn: segyconverter_notebooks
+  timeoutInMinutes: 30
+  displayName: Docker build test
+  pool:
+    name: deepseismicagentpool
+  steps:
+  - bash: |
+      set -e
+      echo "build docker"
+      pwd
+      cd docker
+      pwd
+      docker build -t seismic-deeplearning . 
```

Comment:
I need to check this, but what happens on failure, does Docker throw an exit status? Need to check that it actually fails in the AML pipeline if there's a problem